experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -124,6 +128,10 @@ export const ROUTES: Routes = [
 		RegionsTableComponent,
 		RegionDetailComponent,
 		CacheGroupDetailsComponent,
+		StatusesTableComponent,
+		StatusDetailsComponent,
+		TypesTableComponent,
+		TypeDetailComponent,
```

Comment:
these lines duplicate the "Types" component declarations (which appear below).

Reply:
removed